src/button/Button.ts

```diff
@@ -70,6 +70,30 @@ export default class Button extends ButtonBase<ButtonProperties> {
 	private _onTouchEnd (event: TouchEvent) { this.properties.onTouchEnd && this.properties.onTouchEnd(event); }
 	private _onTouchCancel (event: TouchEvent) { this.properties.onTouchCancel && this.properties.onTouchCancel(event); }
 
+	protected getContent() {
```

Comment:
It may be nice to embrace ES getters if no arguments are passed to these:

```ts
get content() {
    return this.children;
}
```

Reply:
i'd say this is overkill @bitpshr especially if the `getter` ends up doing more than simply getting.

Reply:
Fair.